be/src/runtime/exec_env.cpp

```diff
@@ -233,6 +236,25 @@ const std::string& ExecEnv::token() const {
     return _master_info->token;
 }
 
+void ExecEnv::add_rf_event(const TUniqueId& query_id, int filter_id, const std::string& network, std::string&& msg) {
```

Comment:
Why do we need `&&` here? `strings::Substitule` will create new string any way, and I don't see any benefit to use move operator here. Could you use `const string& msg`

Reply:
use std::string&& to avoid cpy inside of RuntimeFilterCache::add_rf_event. so it is better to use rvalue ref than const&.

Reply:
I know, but you are using `Strings::substitue` underneathe, and there are copy operation.